### PR TITLE
Support images which dont require authentication (no pullSecret required)

### PR DESCRIPTION
**Pull request by sjhx**
to resolve https://github.com/IBM/portieris/issues/123

### CODE REVIEW

**Comment by sjhx** [comment]
This has been tested manually.
Should be tested in e2e. 
The unit test in imagePolicy_test.go connects to external registries, should create an httptest.Server which actls like a registry.

**Comment by sjhx** [comment]
I expect unit test issues to be addressed as part the work on #238

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 # v-next
 ##
 
+# v0.9.6
+## TBD
+* support verifying image which dont require pull secrets ([#123](https://github.com/IBM/portieris/issues/123))
```
Comment (brackendawson) [inline CHANGELOG.md:11]: *images

Comment (sjhx) [reply CHANGELOG.md]: thanks

**Review by brackendawson — APPROVED** [review]

**Pull request merged**